web/components/admin/TextField.tsx

```diff
@@ -52,15 +55,30 @@ export const TextField: FC<TextFieldProps> = ({
   type,
   useTrim,
   value,
+  isAdminPwdField,
 }) => {
+  const [hasPwdChanged, setHasPwdChanged] = useState(false);
```

Comment:
Since `TextField` is used for all kinds of text input, not just passwords, could we use generic naming of things? Maybe `hasChanged` or `hasContentChanged` or something so it doesn't confuse somebody who's looking at the code later and wondering what a password has to do with it.

Reply:
I named them `hasPwdChanged` and `setHasPwdChanged` because they are only for the Admin Password input field and are password specific. However, I can change them to more generic names like `hasChanged` or `hasContentChanged`.  This `useState` makes the submit button for the Admin Password input field visible from the hidden state whenever the user types something inside the Admin Password input field. Also, we can make the submit button for the Admin Password always visible and remove this specific `useState`.